conda_smithy/configure_feedstock.py

```diff
@@ -1198,6 +1199,16 @@ def render_drone(jinja_env, forge_config, forge_dir, return_metadata=False):
     )
 
 
+def render_actions(jinja_env, forge_config, forge_dir, render_info=None):
+    # this file was copied over w/ all of the feedstock content
+    # we have to remove it if the actions are not on
+    if not forge_config.get("bot", {}).get("automerge", False):
```

Comment:
Can you add `bot: automerge: False` as a default in the dictionary in Line 1329?

Reply:
done!

Reply:
```suggestion
    if not forge_config["bot"]["automerge"]:
```

Reply:
Does the code always put in the defaults? Then yes this should work.

Reply:
Ack this won't work with the test I wrote.

Reply:
done!